lib/tasks/scanning.rake

```diff
@@ -1,3 +1,5 @@
+require 'bundler/audit/cli'
```

Comment:
[thought] hmmm, I think your `if env` check from before was actually needed, as I think this will break loading rake tasks in production, not just break when trying to run these tasks. In the past, I've defended against that with something like:

```
begin
  require 'development_gem'
rescue LoadError
  # no-op, probably in a production environment
end
```

That way you don't need to go back and update checks if you move environments a thing is available in

Reply:
yep, you're totally right - just confirmed that behavior locally. thanks for the suggestion!